README.md

```diff
@@ -1,6 +1,12 @@
 # Task list pattern - plugin for the GOV.UK Prototype Kit
 
-This plugin provides the code to use the GOV.UK Task list pattern in prototypes
+This plugin is no longer maintained.
+
+If you are using GOV.UK Prototype Kit version XX or above, you should use the [task list component](https://design-system.service.gov.uk/components/task-list) from the GOV.UK Design System instead.
```

Comment:
TODO: add the version number of the Prototype Kit which contains v5 of `govuk-frontend`.